ast/meta.go

```diff
@@ -2,11 +2,35 @@ package ast
 
 import (
 	"fmt"
-
 	"github.com/VirusTotal/gyp/pb"
 	"github.com/golang/protobuf/proto"
 )
 
+const (
+	lowerhex = "0123456789abcdef"
+)
+
+func toASCII(s string) string {
+	ascii := make([]byte, 0)
```

Comment:
Maybe we should initialize the capacity of `ascii` to the length of `s`?

Reply:
Done.